examples/vite/src/AppSettings/AppSettings.scss

```
@@ -1044,8 +1046,14 @@
 
   .app__settings-modal__tab-body {
     display: flex;
+    flex: 1 1 auto;
     flex-direction: column;
     gap: var(--str-chat__spacing-xl);
+    min-height: 0;
+    // Tall tabs (e.g. Emoji Picker) would otherwise clip under the modal's
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)